content/team/Andrei Smoleu.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Andrei Smoleu
+job_title: Project Manager, Industry Expert
+description: In VibroBox team Andrei is responsible for designing and manufacturing of VibroBox hardware units and accessory, project management, technology testing and certification.
```

Comment:
responsible for equipment design and unit manufacturing, project management, ...